nameko_sqlalchemy/pytest_fixtures.py

```diff
@@ -36,6 +36,24 @@ def db_url(request):
 
 
 @pytest.fixture(scope='session')
+def create_engine_kwargs():
```

Comment:
In that case, what about `db_engine_options` and then we follow the same pattern here and it's more explicit?